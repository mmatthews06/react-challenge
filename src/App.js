import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Button from '@material-ui/core/Button';
import IconButton from '@material-ui/core/IconButton';
import MenuIcon from '@material-ui/icons/Menu';
import PersonIcon from '@material-ui/icons/Person';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },
  menuButton: {
    marginRight: theme.spacing(2),
  },
  title: {
    flexGrow: 1,
  },
  menuItem: {
    color: 'white',
  },
}));

function App() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <AppBar position="static">
        <Toolbar>
          <IconButton edge="start" className={classes.menuButton}>
            <MenuIcon className={classes.menuItem} fontSize="large" />
          </IconButton>
          <Typography variant="h6" className={classes.title}>
           Q2 Solutions
          </Typography>
          <Button>
            <PersonIcon className={classes.menuItem} fontSize="large" />
          </Button>
          <span className={classes.menuItem}>
            q12355@quintiles.net
          </span>
        </Toolbar>
      </AppBar>
    </div>
  );
}

export default App;
