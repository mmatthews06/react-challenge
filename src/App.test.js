/* global describe, expect, it */
import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  it('should match a snapshot', () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
