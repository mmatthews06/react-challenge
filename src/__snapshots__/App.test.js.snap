// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match a snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <WithStyles(ForwardRef(IconButton))
        className="makeStyles-menuButton-2"
        edge="start"
      >
        <MenuIcon
          className="makeStyles-menuItem-4"
          fontSize="large"
        />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-title-3"
        variant="h6"
      >
        Q2 Solutions
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Button))>
        <PersonIcon
          className="makeStyles-menuItem-4"
          fontSize="large"
        />
      </WithStyles(ForwardRef(Button))>
      <span
        className="makeStyles-menuItem-4"
      >
        q12355@quintiles.net
      </span>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
  <div
    className="makeStyles-content-5"
  >
    <Dashboard />
  </div>
</div>
`;
